yt_dlp/extractor/arte.py

```diff
@@ -95,6 +95,24 @@ class ArteTVIE(ArteTVBaseIE):
         )?
     ''')
 
+    __GEO_COUNTRIES = {
+        # obtained by exhaustive testing
+        # with both 'BE' sometimes works, sometimes doesn't
+        'EUR_DE_FR': frozenset((
+            'AT', 'BL', 'CH', 'DE', 'FR', 'GF', 'GP', 'LI',
+            'MC', 'MF', 'MQ', 'NC', 'PF', 'PM', 'RE', 'WF',
+            'YT',
+        )),
+        'SAT': frozenset((
+            'AD', 'AT', 'AX', 'BG', 'BL', 'CH', 'CY', 'CZ',
+            'DE', 'DK', 'EE', 'ES', 'FI', 'FR', 'GB', 'GF',
+            'GR', 'HR', 'HU', 'IE', 'IS', 'IT', 'KN', 'LI',
+            'LT', 'LU', 'LV', 'MC', 'MF', 'MQ', 'MT', 'NC',
+            'NL', 'NO', 'PF', 'PL', 'PM', 'PT', 'RE', 'RO',
+            'SE', 'SI', 'SK', 'SM', 'VA', 'WF', 'YT',
+        )),
```

Comment:
Speaking of which, what is this? 

https://github.com/yt-dlp/yt-dlp/blob/926ccc84ef91498f3147b07d15eb5f40cd070471/yt_dlp/utils.py#L4188

It’s not an ISO 3166 code, and my GeoIP database (the last freely downloadable public version of GeoLite2) locates the IP block in Norway.

Reply:
`Asia/Pacific Region`. It's not in the ISO, but is used for IP blocks (similar to `EU`)

Reply:
`ISO3166Utils.short2full` doesn’t have it either, which causes crashes when `GeoRestrictedError(countries=['AP'])` is raised.

Reply:
So `EU` and `AP` should be added to (the end of ?) `ISO3166Utils._country_map`?